.github/workflows/BuildImage.yml

```diff
@@ -5,8 +5,8 @@ on: [push, pull_request_target, workflow_dispatch]
 env:
   GITHUB_REPO: "linuxserver/docker-mods" #don't modify
   ENDPOINT: "linuxserver/mods" #don't modify
-  BASEIMAGE: "replace_baseimage" #replace
-  MODNAME: "replace_modname" #replace
+  BASEIMAGE: "netbox" #replace
+  MODNAME: "slurpit" #replace
   MULTI_ARCH: "true" #set to false if not needed
```

Comment:
this should be false as the Dockerfile is not arch specific

Reply:
updated